flake.lock

```diff
@@ -0,0 +1,11 @@
+{
+    "inputs": {
+        "nixpkgs": {
+            "inputs": {},
+            "narHash": "sha256-4p9gqMJTVmyH7eSq1euGZWusldWoBfKfi/VwZEPL2Mw=",
+            "originalUrl": "nixpkgs",
+            "url": "github:nrdxp/nixpkgs/1e232452ed4c57e9968f4342d9ba044d36403928"
```

Comment:
I assume there is no way to import the users nixpkgs instead of this?

Reply:
Also this should point to NixOS instead of your fork.

Reply:
Ah you fixed some stuff: https://github.com/nrdxp/nixpkgs/commits/1e232452ed4c57e9968f4342d9ba044d36403928

Reply:
Well I didn't mean to use my fork. I have my fork pinned globally so it got set since I didn't specify the url. You can easily replace it in by specifying a flake uri explicitly, via `inputs.nixpkgs.url = "github:owner/nixpkgs/branch`, and then running `nix flake update`. I fix this and set it to the default upstream since it was a mistake to begin with.